pkg/driver/utils.go

```diff
@@ -106,3 +107,29 @@ func getCapacity(cvc *apisv1.CStorVolumeConfig) string {
 	qCap := cvc.Spec.Capacity[corev1.ResourceStorage]
 	return qCap.String()
 }
+
+func getVolumeCondition(vol *apisv1.CStorVolume) *csi.VolumeCondition {
+	condition := &csi.VolumeCondition{}
+	if vol.Status.Phase != apisv1.CVStatusHealthy {
+		condition.Abnormal = true
+	}
+
```

Comment:
This CStorVolume status can be stale when the target pod is not in an available state... Can we update the status of CStorVolume when a process in the container gets SIGINT/SIGKILL signal?